docs/code-examples/entity_path.cpp

```diff
@@ -14,4 +14,7 @@ int main() {
         rerun::new_entity_path({"world", std::to_string(42), "unescaped string!"}),
         rerun::TextDocument("This entity path was provided as a list of unescaped strings")
     );
+
+    assert(rerun::escape_entity_path_part("my string!") == R"(my\ string\!)");
+    assert(rerun::new_entity_path({"world", "42", "my string!"}) == R"(/world/42/my\ string\!)");
```

Comment:
why are the string literals in parenthesis?

Reply:
That's the silly syntax C++ uses for raw string literals: https://en.cppreference.com/w/cpp/language/string_literal

Reply:
oh, didn't realize. that's sad